assets/src/components/nav/topNavMobile.tsx

```diff
@@ -0,0 +1,153 @@
+import React, { useContext } from "react"
+import { useLocation, Link, NavLink } from "react-router-dom"
+import { StateDispatchContext } from "../../contexts/stateDispatchContext"
+import { displayHelp } from "../../helpers/appCue"
+import {
+  closeIcon,
+  hamburgerIcon,
+  logoIcon,
+  questionMarkIcon,
+  refreshIcon,
+  settingsIcon,
+  speechBubbleIcon,
+} from "../../helpers/icon"
+import { toggleMobileMenu, toggleNotificationDrawer } from "../../state"
+import NotificationBellIcon from "../notificationBellIcon"
+import { currentRouteTab } from "../../models/routeTab"
+import { openDrift } from "../../helpers/drift"
+import { reload } from "../../models/browser"
+
+const topNavMobile = (): JSX.Element => {
+  const location = useLocation()
+
+  const [state, dispatch] = useContext(StateDispatchContext)
+
+  const { routeTabs, notificationDrawerIsOpen, mobileMenuIsOpen } = state
+
+  const bellIconClasses = notificationDrawerIsOpen
+    ? ["m-top-nav__notifications-icon", "m-top-nav__notifications-icon--active"]
+    : ["m-top-nav__notifications-icon"]
+
+  let tabName = "Untitled"
+  const showTabName = location.pathname === "/"
+  const currentTab = currentRouteTab(routeTabs)
+  if (showTabName && currentTab) tabName = currentTab.presetName || "Untitled"
```

Comment:
Given that we now have logic both here and in `LadderPage` for defaulting the display name of a tab to "Untitled" if it isn't saved as a preset, I wonder if we could split some of this out into a small helper function in `models/routeTab.ts`.

Reply:
added this in 5760972